File: simplicrypto/src/main/java/com/trayis/simplicrypto/SecurityConstants.java
package com.trayis.simplicrypto;

public @interface SecurityConstants {

    String STORE_NAME = "security-mStore";

    String PROVIDER_ANDROID_KEY_STORE = "AndroidKeyStore";

    char[] PASSWORD = "password".toCharArray();

    String ALGORITHM_AES = "AES";

    String BLOCK_MODE_CBC = "CBC";

    String PADDING_PKCS_7 = "PKCS7Padding";

    String AES_CBC_PKCS7PADDING = "AES/CBC/PKCS7Padding";

    String TRANSFORMATION_SYMMETRIC = AES_CBC_PKCS7PADDING;
}

File: simpliCrypto/src/main/java/com/trayis/simplicrypto/SecurityConstants.kt
package com.trayis.simplicrypto

class SecurityConstants {
    companion object {

        const val STORE_NAME = "security-mStore"

        const val PROVIDER_ANDROID_KEY_STORE = "AndroidKeyStore"

        val PASSWORD = "password".toCharArray()

        const val ALGORITHM_AES = "AES"

        const val BLOCK_MODE_CBC = "CBC"

        const val PADDING_PKCS_7 = "PKCS7Padding"

        const val AES_CBC_PKCS7PADDING = "AES/CBC/PKCS7Padding"

        const val TRANSFORMATION_SYMMETRIC = AES_CBC_PKCS7PADDING
    }
}
